frontend/language/src/nb.json

```
@@ -857,7 +863,53 @@
   "resourceadm.left_nav_bar_policy": "Tilgangsregler",
+  "resourceadm.listadmin_add_to_list": "Legg til i liste",
+  "resourceadm.listadmin_back": "Tilbake til dashboard",
+  "resourceadm.listadmin_confirm_create_list": "Opprett enhetsliste",
+  "resourceadm.listadmin_create_list": "Opprett ny enhetsliste",
+  "resourceadm.listadmin_create_list_description": "Velg navn og id for listen. Navnet er for intern bruk, og id er foreslått basert på navnet du skriver men kan redigeres om du ønsker en annen. Navn kan endres senere, mens id kan ikke endres.",
+  "resourceadm.listadmin_create_list_header": "Lag ny enhetsliste i {{env}}",
+  "resourceadm.listadmin_delete_list": "Slett liste",
+  "resourceadm.listadmin_delete_list_description": "Vil du slette denne listen?",
+  "resourceadm.listadmin_delete_list_header": "Bekreft sletting av liste",
+  "resourceadm.listadmin_empty_list": "Listen inneholder ingen enheter",
+  "resourceadm.listadmin_empty_name": "<navn ikke funnet>",
+  "resourceadm.listadmin_header": "Administrer enhetslister i ressurseierstyrt rettighetsregister",
+  "resourceadm.listadmin_identifier_conflict": "En enhetsliste med denne id-en finnes fra før",
+  "resourceadm.listadmin_list_description": "Beskrivelse",
+  "resourceadm.listadmin_list_description_description": "Her kan du beskrive listen",
+  "resourceadm.listadmin_list_detail_header": "Administrer enhetsliste",
+  "resourceadm.listadmin_list_id": "Liste-id",
+  "resourceadm.listadmin_list_id_description": "Liste-id brukes for å identifisere listen, og kan ikke endres",
+  "resourceadm.listadmin_list_name": "Listenavn",
+  "resourceadm.listadmin_list_name_description": "Gi listen et beskrivende navn, f.eks \"Godkjente banker\"",
+  "resourceadm.listadmin_list_organizations": "Registrerte enheter",
+  "resourceadm.listadmin_list_organizations_description": "Enheter i denne listen vil ha tilgang til ressursen",
+  "resourceadm.listadmin_load_list_error": "Kunne ikke laste enhetslister",
+  "resourceadm.listadmin_navn": "Navn",
+  "resourceadm.listadmin_orgnr": "Orgnr",
+  "resourceadm.listadmin_parties": "Enheter",
+  "resourceadm.listadmin_party": "Enhet",
+  "resourceadm.listadmin_remove_from_list": "Fjern fra liste",
+  "resourceadm.listadmin_resource_header": "Ressurseierstyrt rettighetsregister for {{resourceTitle}} - {{env}}",
+  "resourceadm.listadmin_resource_list_checkbox_header": "Alle enheter og underenheter i valgte liste(r) kan bruke ressursen",
+  "resourceadm.listadmin_search": "Søk for å legge til ny enhet eller underenhet",
+  "resourceadm.listadmin_search_add_more": "Legg til flere enheter",
+  "resourceadm.listadmin_search_first": "Første",
+  "resourceadm.listadmin_search_last": "Siste",
+  "resourceadm.listadmin_search_next": "Neste",
+  "resourceadm.listadmin_search_no_parties": "Fant ingen enheter",
+  "resourceadm.listadmin_search_no_sub_parties": "Fant ingen underenheter",
+  "resourceadm.listadmin_search_paging": "Viser {{fra}}-{{til}} av {{total}}",
```

I belive the variables should be on english 😄 
```suggestion
  "resourceadm.listadmin_search_paging": "Viser {{from}}-{{to}} av {{total}}",
```